"0","import pandas as pd
import segFunctions

# runs the csv_from_excel function:
# creates csv for segmentation experiment lists
segFunctions.csv_from_excel('Segmentation_Experimental_Item_Setup.xlsx','All_Experiment_Lists','./data/original_data/exp_files/expLists_all.csv')

# creates csv for segmentation practice lists
segFunctions.csv_from_excel('Segmentation_Experimental_Item_Setup.xlsx','All_Practice_Lists','./data/original_data/exp_files/pracLists_all.csv')

# creates csv for syllabification experiment lists
segFunctions.csv_from_excel('Segmentation_Experimental_Item_Setup.xlsx','Syllabification_Lists','./data/original_data/exp_files/sylexpLists_all.csv')

# creates csv for syllabification practice lists
segFunctions.csv_from_excel('Segmentation_Experimental_Item_Setup.xlsx','Syllabification_Practice','./data/original_data/exp_files/sylpracLists_all.csv')"
"2","Traceback (most recent call last):
  File ""/var/folders/c7/hfz1q3md61v2h5rpl1l_7_cc0000gn/T/RtmpnRdKSn/chunk-code-45441a65e83.txt"", line 1, in <module>
    import pandas as pd
ImportError: No module named pandas
"
